cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -77,3 +80,20 @@ func nodeRegister(
 	// If gRPC server is gracefully shutdown, exit
 	os.Exit(0)
 }
+
+func buildSocketPath(csiDriverName string) string {
+	return fmt.Sprintf("/registration/%s-reg.sock", csiDriverName)
+}
+
+func removeRegSocket(csiDriverName string) {
+	sigc := make(chan os.Signal, 1)
+	signal.Notify(sigc, syscall.SIGTERM)
```

Comment:
Any ideas whether this will work on windows ? @ddebroy - any ideas ?
If not can you please move this to linux utils ?

Reply:
if someone confirms it wont work in windows i will move it linunx_utils

Reply:
cc @jingxu97 as well

Reply:
Notifications on `syscall.SIGTERM` does work for Windows: https://golang.org/pkg/os/signal/#hdr-Windows. So LGTM from a Windows perspective. cc @jingxu97